README.md

```diff
@@ -53,6 +62,8 @@ There are lots of bugs. Check issues on the code repo.
 ### Credits
 - Reverse engineered by GalaXyHaXz in 2018
 - Switch Port by MVG in 2019
+- Control Improvements and bug fixes for Switch Port by rsn8887 in 2019
+- Control Improvements and bug fixes for Switch Port by erfg12 in 2019
```

Comment:
erfg12 is already in the credits as Jacob Fliss, maybe make his name a link to his profile?

Reply:
Ok perfect I will fix this.

Reply:
fixed with 
87a3012